lib/components/Inline/Inline.treat.ts

```diff
@@ -2,11 +2,23 @@ import { style } from 'treat';
 
 import { makeResponsiveStyle } from '../../utils';
 
-export const root = style({
-	display: 'flex',
-	justifyContent: 'flex-start',
-	flexWrap: 'wrap',
-});
+const flexShared = style(
+	{
+		display: 'flex',
+	},
+	'flexShared',
+);
+
+export const root = [
+	flexShared,
+	style(
+		{
+			justifyContent: 'flex-start',
+			flexWrap: 'wrap',
+		},
+		'root',
+	),
+];
 
 const alignRules = {
```

Comment:
Since this is going to be a common use case do you think it's worth exposing other divider alignment options (center, felx-end and stretch) so we don't need to use custom styles for those